vagrant/Makefile

```diff
@@ -1,5 +1,4 @@
 INVENTORY := ./vagrant_ansible_inventory
-VSSH := ssh -F ansible/ssh-config-setup -i /root/.vagrant.d/insecure_private_key
```

Comment:
I still don't really like doing `vagrant ssh`.
Instead, you could just use:
```
VSSH := ssh -F ssh-config-host
```
That should just work. Created #54 as an alternative to see if the CI passes with it.